legal-api/src/legal_api/services/authz.py

```diff
@@ -306,7 +298,7 @@ def has_roles(jwt: JwtManager, roles: List[str]) -> bool:
             },
             'dissolution': {
                 'voluntary': {
-                    'legalTypes': ['CP', 'BC', 'BEN', 'CC', 'ULC', 'LLC', 'SP', 'GP'],
+                    'legalTypes': ['CP', 'BC', 'BEN', 'CC', 'ULC', 'SP', 'GP'],
                     'blockerChecks': {
                         'warningTypes': [WarningType.MISSING_REQUIRED_BUSINESS_INFO],
                         'business': [BusinessBlocker.DEFAULT]
```

Comment:
need to add business blocker to exclude voluntary dissolutions when business is not in good standing